.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "presets": ["env"],
+  "sourceMap": "inline",
+  "plugins": [
+    "transform-object-rest-spread"
```

Comment:
@iondrimba is this plugin the same of here https://github.com/babel/babel-preset-env/blob/master/data/plugin-features.js#L77? If yes, `env` already contains that.

Reply:
@eduardojmatos Tested, they are the same 👍